xbmc/guilib/GUIIncludes.h

```diff
@@ -52,7 +52,17 @@ class CGUIIncludes
   const INFO::CSkinVariableString* CreateSkinVariable(const std::string& name, int context);
 
 private:
+  enum RESOLVE_PARAMS_RESULT
```

Comment:
I usually use
```
enum Foo
{
  FooNone,
  FooThis,
  FooThat
};
```
but I don't think we really have a guideline on this. It's just most in line with everything else.